### ISSUE TITLE
vscode powershell extension - highlight text and "run selection" results in System.ArgumentOutOfRangeException

### ISSUE THREAD

**Issue by MattSntap** [PowerShell/PSReadLine#2840]
Few factors that may be at play (brand new laptop), but I did notice the vscode powershell extension was updated this morning.  Though I have not performed these specific actions on the new laptop yet.

Error appears in psversion 5.1.18362.1714 ~~and 7.1.4~~.

I normally can highlight the entire 1026 lines in this script and "run selection" without issue.

PSReadline version 2.0.0

Text from console:

```powershell
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 200 Keys:
 o p e r t y N a m e Space " s i z e _ t o _ c h a n g e M B " Space Space Space Space Space Space Space - N o t e P r o p e r t y V a l u e Space $ ( [ m a t h ] : : R o u n d ( $ s i z e _ t o _ c h a n g e M B , Space 1 ) ) Enter
 Space Space Space Space Space Space Space Space Space Space Space Space } Enter
 Space Space Space Space Space Space Space Space } Enter
 Enter
 Space Space Space Space Space Space Space Space r e t u r n Space $ a g g r e g a t e s Space | Space S o r t - O b j e c t Space - P r o p e r t y Space u s e d R a t i o Space - D e s c e n d i n g Enter
 Space Space Space Space } Enter
 } Enter


Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -58.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

**Comment by daxian-dbw** [comment]
This issue was already fixed (see #1306). Please upgrade to the [2.1.0 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.1.0) from PowerShell Gallery.
See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.

**Comment by MattSntap** [comment]
> This issue was already fixed (see #1306). Please upgrade to the [2.1.0 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.1.0) from PowerShell Gallery.
> See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.

just came back to close this for the same, thanks!
